repos.d/nix/nur.yaml

```diff
@@ -0,0 +1,34 @@
+###########################################################################
+# NUR
+###########################################################################
+- name: nur
+  sortname: nix-unstable~nur  # place right after Nix unstable
+  type: repository
+  desc: NUR combined
+  statsgroup: nix
+  family: nix
+  ruleset: [nix]
+  color: '7eb2dd'
+  update_period: 24h
```

Comment:
We can update it more often (e.g. remove this line). AUR is a problem because there's no single file with all the info and we have to paginate through the API, this is likely not the cause here.

Reply:
AFAIK the info is regenerated once a day, but I agree: Let's keep artificial delays low.